pages/apps/leads-management/crm-editing/setup-status/index.js

```diff
@@ -89,8 +96,8 @@ export default function Motivation() {
                 />
 
                 <p>
-                  From here you can change the status list, Default Open Status
-                  & Default Close Status.
+                From here you can change the status, add new statuses, or remove statuses from the status list.You can also change the Default Open Status
```

Comment:
Make it as `add new status, or remove status from the status list`. Basically replace statuses with "status".

Reply:
Ideally it should be like this :

`From here you can change the status, (add new status, or remove status from the status list).`